source/codegen/client_helpers.py

```diff
@@ -124,6 +124,8 @@ def _get_cpp_type_for_protobuf_type(protobuf_type: str) -> str:
 
     if protobuf_type in PROTOBUF_TYPE_TO_CPP_TYPE:
         cpp_type = PROTOBUF_TYPE_TO_CPP_TYPE[protobuf_type]
+        if cpp_type in ["string"]:
```

Comment:
This is to fix up the generated _client.h and _client.cpp files. Found out in some recent update to protobuf that `string` is no longer defined in `::google::protobuf`. You can see the issue raised [here](https://github.com/protocolbuffers/protobuf/issues/9943) but apparently it was intentional and the recommendation is just to use `std::string` instead which is what this update is about.